test/cli.test.js

```diff
@@ -100,5 +100,31 @@ test('cli', (t) => {
     t.tearDown(() => child.kill())
   })
 
+  t.test('uses specified timestampKey', (t) => {
+    t.plan(1);
+    const env = { TERM: 'dumb' }
+    const child = spawn(process.argv[0], [bin, '--timestampKey', '@timestamp'], { env })
+    child.on('error', t.threw)
+    child.stdout.on('data', (data) => {
+      t.is(data.toString(), '[1522431328992] INFO : hello world\n    @timestamp: 1522431328992\n')
+    })
+    const logLine = '{"level":30,"@timestamp":1522431328992,"msg":"hello world"}\n'
+    child.stdin.write(logLine)
+    t.tearDown(() => child.kill())
+  });
+
+  t.test('uses an ignored timestampKey', (t) => {
+    t.plan(1);
+    const env = { TERM: 'dumb' }
+    const child = spawn(process.argv[0], [bin, '--timestampKey', '@timestamp', '--ignore', '@timestamp'], { env })
+    child.on('error', t.threw)
+    child.stdout.on('data', (data) => {
+      t.is(data.toString(), '[1522431328992] INFO : hello world\n')
+    })
+    const logLine = '{"level":30,"@timestamp":1522431328992,"msg":"hello world"}\n'
+    child.stdin.write(logLine)
+    t.tearDown(() => child.kill())
+  });
```

Comment:
Thank you. These look good for covering what the CLI does. I'd also like to see the behavior covered in the factory tests, e.g.:

https://github.com/pinojs/pino-pretty/blob/62a5f01d31aef4ca8ff69a08180080bd88506c2c/test/basic.test.js#L603-L615

Reply:
No problem, I've added those too.